package-lock.json

```diff
@@ -10949,9 +10949,9 @@
       }
     },
     "node_modules/@types/d3-array": {
```

Comment:
@munteannatan  could you do whatever additional validation you think is necessary for the d3 updates? This could include
-  manual testing of the storybook build
-  looking at release notes for breakages or new features we want to use or bug fixes that allow us to remove workarounds

Note: this is just a package-lock update, meaning it affects the versions used by the repo's pipeline but it doesn't affect the versions that clients like SLE are required to install

Also note: the d3 updates are just for `@types` packages, so if you think they need no additional validation beyond the build passing, that's fine by me.

Reply:
The d3 updates are just backward compatible bug fixes and patches. The storybook build is behaving as expected when zooming and using the scaling from d3.